ios/Runner/Info.plist

```diff
@@ -53,14 +53,14 @@
         <dict>
             <key>CFBundleURLSchemes</key>
             <array>
-                <string>fb567334037481721</string>
+                <string>FB_LOGIN_PROTOCOL_SCHEME</string>
             </array>
         </dict>
 		<!-- Google authentication config -->
 		<dict>
 			<key>CFBundleURLSchemes</key>
 			<array>
-                <string>com.googleusercontent.apps.377832615490-d5i0e2guo62fja096ugpnjak12sleo8m</string>
+                <string>REVERSED_CLIENT_ID</string>
```

Comment:
REVERSED ?

Reply:
This is the way cliend_id is present in google services file with this technical name. @tachyons